.eslintignore

```diff
@@ -1,3 +1,2 @@
-node_modules/
 coverage/
```

Comment:
I meant the whole `.eslintignore` file is not needed. @segayuu and I recently [started](https://github.com/hexojs/hexo-renderer-less/pull/25#discussion_r311563855) to [remove](https://github.com/hexojs/hexo-util/pull/73/commits/f5b8842c398ad0664537ce8f9388ada5d9f55d2f) this file from hexo repos.